app/renderer/components/styles/global.js

```diff
@@ -327,6 +328,13 @@ const globalStyles = {
         background: '#ccc'
       }
     }
+  },
+
+  // TODO (Suguru): move them to payment.js after style refactoring is done
```

Comment:
ok thanks. also pls move things out of that most you can. I think that file should be removed